pkg/pillar/cmd/downloader/downloader.go

```diff
@@ -34,40 +32,17 @@ const (
 
 // Go doesn't like this as a constant
```

Comment:
Hmm.. does this comment still make sense??

Reply:
Not at all. I am removing it.